### ISSUE TITLE
Blank card

### ISSUE THREAD

**Issue by npawelek** [aukedejong/lovelace-windrose-card#27]
I've updated to the newest release and replaced with the new card definition. After doing so, the card refuses to render anything other than a blank box. I've tried the webapp, mobile app, and an incognito window to rule out a cache problem. Any thoughts as what the issue could be?

There are some frontend errors in the logs after a clean restart of home assistant.

```
2023-06-09 16:08:04.291 ERROR (MainThread) [frontend.js.latest.202306080] :0:0 ResizeObserver loop completed with undelivered notifications.
2023-06-09 16:08:04.302 ERROR (MainThread) [frontend.js.latest.202306080] :0:0 ResizeObserver loop completed with undelivered notifications.
2023-06-09 16:08:13.620 ERROR (MainThread) [frontend.js.latest.202306080] :0:0 ResizeObserver loop completed with undelivered notifications.
2023-06-09 16:08:24.218 ERROR (MainThread) [frontend.js.latest.202306080] :0:0 ResizeObserver loop completed with undelivered notifications.
2023-06-09 16:08:26.285 ERROR (MainThread) [frontend.js.latest.202306080] :0:0 ResizeObserver loop completed with undelivered notifications.
```

I'm running home assistant core version 2023.6.1 with the following config for the windrose card:

```
type: custom:windrose-card
hours_to_show: 24
max_width: 400
refresh_interval: 300
windspeed_bar_location: bottom
windspeed_bar_full: false
wind_direction_entity:
  entity: sensor.weather_station_wind_direction_degrees
  direction_unit: degrees
  use_statistics: false
  direction_compensation: 0
windspeed_entities:
  - entity: sensor.weather_station_wind_speed
    name: 'Average windspeed'
    speed_unit: mph
    use_statistics: false
output_speed_unit: mph
speed_range_beaufort: true
speed_range_step: 5
speed_range_max: 30
windrose_draw_north_offset: 17
cardinal_direction_letters: NESW
matching_strategy: direction-first
center_calm_percentage: true
```

**Comment by npawelek** [comment]
After another subsequent restart of home assistant, I don't see the frontend errors in the logs, but the card is still blank.

**Comment by nameless66** [comment]
Same here. Protocol says:
Logger: frontend.js.latest.202306080
Source: components/system_log/__init__.py:257 
First occurred: 12:38:54 (1 occurrences) 
Last logged: 12:38:54

:0:0 Script error.

**Comment by olivierouellet** [comment]
If you use Chrome and go to View > Developer > Developer tools , do you see any error?
Might be the same as issue #26 .

**Comment by npawelek** [comment]
Yes, it looks like this is the case. My weather station data is via SDR and is temporarily undefined/unavailable until the first polling interval after the restart.

```
Uncaught (in promise) TypeError: this.directionSpeedRangeCounts[windDirectionIndex] is undefined
    add https://hass.domain.tld/hacsfiles/lovelace-windrose-card/windrose-card.js?hacstag=591270696100:1319
    addWindMeasurements https://hass.domain.tld/hacsfiles/lovelace-windrose-card/windrose-card.js?hacstag=591270696100:1368
    refreshData https://hass.domain.tld/hacsfiles/lovelace-windrose-card/windrose-card.js?hacstag=591270696100:1599
```

**Comment by npawelek** [comment]
Should we close this and favor issue #26?

**Comment by aukedejong** [comment]
I am not sure it is the same issue. Maybe we should leave it open for now.

Tomorrow evening I am home again. Didn't bring my laptop on a weekend away.

**Comment by npawelek** [comment]
No worries, enjoy the time away!

**Comment by mattfox27** [comment]
im having same issue, using ecowitt whittboy weather station, blank card

**Comment by ildar170975** [comment]
Not sure about my case:
![изображение](https://github.com/aukedejong/lovelace-windrose-card/assets/71872483/90124d88-6094-40ef-930d-069da9fee20d)

![изображение](https://github.com/aukedejong/lovelace-windrose-card/assets/71872483/8ee15590-5ae6-4751-8617-1e86ac61581d)

**Comment by npawelek** [comment]
Looks like the issue is solved with 1.0.2.
